docs/getting-started/concepts/permission.md

```diff
@@ -93,29 +93,51 @@ Permission(
 ```
 
 ## Authorizing Feast clients
-**TODO**
-Initial proposals:
-* Kubernetes RBAC:
+If you would like to leverage the permissions' functionality then `auth` config block should be added to feature_store.yaml. Currently, feast supports oidc and kubernetes rbac authentication/authorization. 
+
+The default configuration if you don't mention the auth configuration is no_auth configuration.
+
+* No auth configuration example:
+```yaml
+project: foo
+registry: "registry.db"
+provider: local
+online_store:
+    path: foo
+entity_key_serialization_version: 2
+auth:
+  type: no_auth
+```
+
+* Kubernetes rbac authentication/authorization example:
+{% hint style="info" %}
+**NOTE**: This configuration will only work if you deploy feast on Openshift or Kubernetes platform.
+{% endhint %}
 ```yaml
-offline_store:
-    type: remote
-    host: localhost
-    port: 8815
-    auth:
-        type: kubernetes
+project: foo
+registry: "registry.db"
+provider: local
+online_store:
+  path: foo
+entity_key_serialization_version: 2
+auth:
+  type: kubernetes
```

Comment:
is this complete example for kubernetes or it required any addional parameters ?

Reply:
it is complete. No configurations as of now. Just reads the token from the fixed location. on the pod.
https://github.com/lokeshrangineni/feast/blob/feature/oidc-doc-unit-test/sdk/python/feast/permissions/client/kubernetes_auth_client_manager.py#L13